dpctl/tensor/libtensor/source/tensor_py.cpp

```diff
@@ -2081,4 +2371,25 @@ PYBIND11_MODULE(_tensor_impl, m)
           [](sycl::device dev) -> std::string {
               return get_default_device_complex_type(dev);
           });
+    m.def(
+        "_tril",
+        [](sycl::queue exec_q, dpctl::tensor::usm_ndarray src,
```

Comment:
For consistency use `src, dst, k, exec_q, depends` order of arguments.

If change this order, we should change it throughout, and in a separate PR.